build.gradle

```diff
@@ -669,7 +674,105 @@ configure(updateBuildScript) {
     description = 'Updates the build script to the latest version'
 }
 
-// Deobfuscation
+// Parameter Deobfuscation
+
+task deobfParams {
+    doLast {
+        checkPropertyExists("channel")
+        checkPropertyExists("mappingsVersion")
+
+        String mcpDir = System.getProperty("user.home") + "/.gradle/caches/minecraft/de/oceanlabs/mcp/mcp_" + channel + "/" + mappingsVersion
```

Comment:
Use the `project.gradle.gradleUserHomeDir` property instead of guessing `~/.gradle/`: <https://docs.gradle.org/6.9.1/dsl/org.gradle.api.invocation.Gradle.html#org.gradle.api.invocation.Gradle:gradleUserHomeDir>